app/src/service_endpoints.h

```diff
@@ -349,42 +250,22 @@ namespace scitt
       .set_forwarding_required(ccf::endpoints::ForwardingRequired::Never)
       .install();
 
-    registry
-      .make_endpoint(
-        "/.well-known/did.json",
-        HTTP_GET,
-        ccf::json_adapter(std::bind(
-          endpoints::get_did_document, service_certificate_index, _1, _2)),
-        {ccf::empty_auth_policy})
-      .install();
-
-    // The DID document used to be served under this endpoint only, until we
-    // added the .well-known path above. As a transitional measure, we keep
-    // both of them around for a bit.
-    registry
-      .make_endpoint(
-        "/scitt/did.json",
-        HTTP_GET,
-        ccf::json_adapter(std::bind(
-          endpoints::get_did_document, service_certificate_index, _1, _2)),
-        {ccf::empty_auth_policy})
-      .install();
-
+    // FIXME: /jwks should return CBOR errors by default
```

Comment:
This is out of the scope of the draft, but I think it'd be nice to provide a CBOR endpoint for this too/instead, and remove that last bit of JSON. There is no CWK though, but there is a COSE Key format: https://www.rfc-editor.org/rfc/rfc8152#page-33

I think the right way to do this is to have an API to get the current ledger CoseKey from CCF (same as the service cert), and to expose that at a `/cose_keys` or similar, and update the transparency config accordingly. This is very much for later though and not something that requires short term attention.